Peripherals/CAN/CAN.c

```diff
@@ -22,7 +22,7 @@
 	 
 	// Set bit rates 
 	CAN1->BTR &= ~(((0x03) << 24) | ((0x07) << 20) | ((0x0F) << 16) | (0x1FF)); 
-	CAN1->BTR |=  (((2-1) & 0x07) << 20) | (((9-1) & 0x0F) << 16) | ((6-1) & 0x1FF);
+	CAN1->BTR |=  (((2-1) & 0x07) << 20) | (((15-1) & 0x0F) << 16) | ((2-1) & 0x1FF);
```

Comment:
we should probably do something about this and add support to be able to change this via defining a constant